include/kodlab_mjbots_sdk/jointMoteus.h

```diff
@@ -0,0 +1,66 @@
+#include joint.h
+#include "kodlab_mjbots_sdk/moteus_protocol.h"
+#include "kodlab_mjbots_sdk/pi3hat_moteus_interface.h"
+
+/**
+ * @brief 
+ * Moteus Joint class that includes all the motor specific vectors. 
+ */
+class JointMoteus:public JointBase{
+    /**
+     * @brief 
+     * 
+     */
+    public:
+        const int can_bus;
+        const int can_id;
+
+        mjbots::moteus::Mode mode_ = ::mjbots::moteus::Mode::kStopped;
+        mjbots::moteus::Pi3HatMoteusInterface::ServoCommand command;
```

Comment:
I'd recommend that on your first pass through you don't mess around with the structs in the next 4 lines of code since they can be confusing and cause bugs if not handled properly.